Add shared / updated column documentation to default views (#168)
ccao-data/data-architecture: dbt/models/shared_columns.md

property. Also used to determine the property class.

Possible values for this variable are:

- `1` = 1 story (`1STRY`)
- `2` = 2 story (`2STRY`)
- `3` = 3 story or more (`3STRY+`)
- `4` = Split level (`SPLT`)
- `5` = 1.5 story (`1.5STRY`)
- `9.9` = Missing (`MSSNG`)
{% enddocs %}

## char_unit_sf

{% docs shared_column_char_unit_sf %}
Square footage of the condominium unit associated with this PIN.

Applies to condos only
{% enddocs %}

## char_use

{% docs shared_column_char_use %}
Single or multi-family use.

Possible values for this variable are:

- `1` = Single-family (`SF`)
- `2` = Multi-family (`MF`)
{% enddocs %}

## char_yrblt

{% docs shared_column_char_yrblt %}
Year the property was constructed
{% enddocs %}

# Cook County

## card

{% docs shared_column_card %}
Sub-unit of a PIN.

For residential properties, cards usually identify each *building*, For
commercial properties, they can identify spaces within the same building.
Cards also serve as the unit of observation for the residential model.

Equivalent to legacy `MLT_CD` (multicode) value
{% enddocs %}

## cdu

{% docs shared_column_cdu %}
Condition/Desirability/Utility code.

> Code representing a any number of seemingly unrelated characteristics

Reviewer: ```suggestion
Code representing any number of seemingly unrelated characteristics
```

of taxable value in Cook County.

All PINs are 14 digits: 2 digits for area + 2 digits for sub area + 2 digits
for block + 2 digits for parcel + 4 digits for the condominium unit/leasehold
{% enddocs %}

## pin10

{% docs shared_column_pin10 %}
First 10 digits of a PIN.

Useful for identifying individual condominium buildings since the last 4
digits of a PIN identifies individual units
{% enddocs %}

## tax_code

{% docs shared_column_tax_code %}
Property tax code.

Identifies the unique combination of taxing districts which impose a levy
on any given property
{% enddocs %}

## township_code

{% docs shared_column_township_code %}
Cook County township code.

See `township_name` for more information. Note that township codes that start
with 7 are City triad townships
{% enddocs %}

## township_name

{% docs shared_column_township_name %}
Cook County township name.

The county is divided into 38 geographic townships. These townships act as
units of work and analysis for the Assessor's Office. Township boundaries are
coincident with triads (triads are made up of townships). Note that townships
can also be units of local government, with their own boards and taxing

> authority. Townships in the City of Chicago are effectively vestigial

Reviewer: spicy

{% enddocs %}

# Proration, Multi-cards, and Landlines

## card_proration_rate

{% docs shared_column_card_proration_rate %}
Proration rate for a card on a given PIN.

Prorated cards split their total value across multiple PINs. An example is
something like a building that crosses multiple PINs
{% enddocs %}

## lline

{% docs shared_column_lline %}
Record number related to land or building "line".
{% enddocs %}

## pin_is_multiland

{% docs shared_column_pin_is_multiland %}
Indicates the PIN has more than one landline.
{% enddocs %}

## pin_is_multicard

{% docs shared_column_pin_is_multicard %}
Indicates a PIN with more than one building (ADU, coach house, etc.).
{% enddocs %}

## pin_is_multilline

{% docs shared_column_pin_is_multilline %}
Indicates a PIN with multiple values for `lline`.
{% enddocs %}

## pin_num_cards

{% docs shared_column_pin_num_cards %}
Number of cards associated with a PIN.

PINs can have multiple cards (buildings) within their area. Multi-card PINs
are valued slightly differently than single-card ones.
{% enddocs %}

## pin_num_landlines

{% docs shared_column_pin_num_landlines %}
Number of landlines associated with a PIN.

> Each landlines is typically associated with an individual land rate i.e.

Reviewer: ```suggestion
Each landline is typically associated with an individual land rate i.e.
```

PINs are occasionally prorated when not all of their value is contained within
their boundaries. For example, a building that lies equally across two PINs
would be prorated to 50%. In this case, the _land_ value of the PIN is not
prorated, but the building value is.
{% enddocs %}

# Sales

## buyer_name

{% docs shared_column_buyer_name %}
Name of property buyer, as listed on deed.

Can be truncated by myDec/IDOR. See Clerk/Recorder of Deeds for full name.
{% enddocs %}

## document_number

{% docs shared_column_document_number %}
Deed number/document number of the sale.

Serves as the unique identifier for each sale. Can be used to lookup more
information on the Clerk/Recorder of Deeds website.
{% enddocs %}

## deed_type

{% docs shared_column_deed_type %}
Numerically-encoded deed type of the sale.

Possible values for this variable include:

- `01` - Warranty
- `02` - Trustee
- `03` - Quit claim
- `04` - Executor
- `05` - Other
- `06` - Beneficiary
- `99` - Unknown
{% enddocs %}

## num_parcels_sale

{% docs shared_column_num_parcels_sale %}
Number of parcels that are part of multisale
{% enddocs %}

## sale_date

{% docs shared_column_sale_date %}
Date of property sale.

> Uses dates from `sale.mydec` prior to 2021. Also defaults to `sale.mydec`

Reviewer: ```suggestion
Uses dates from `sale.mydec` between 2014 and 2021. Also defaults to `sale.mydec`
```

any time `iasworld.sales` and `sale.mydec` disagree.
{% enddocs %}

## sale_is_multisale

{% docs shared_column_sale_is_multisale %}
Indicates whether or not a sale involved multiple parcels
{% enddocs %}

## sale_key

{% docs shared_column_sale_key %}
iasWorld internal sale identifier
{% enddocs %}

## sale_price

{% docs shared_column_sale_price %}

> Fair market sale price of a PIN, as recorded on the deed.

Reviewer: We don't actually know that a random value of sale_price is fair market, do we?

Author: True, I'll update the language.
